src/apprt/gtk/App.zig

```diff
@@ -1438,6 +1440,16 @@ fn newWindow(self: *App, parent_: ?*CoreSurface) !void {
     try window.newTab(parent_);
 }
 
+fn closeWindow(_: *App, target: apprt.action.Target) void {
```

Comment:
Might as well make this performable and do a boolean return value from this and `performAction`. No reason not to.